dipy/workflows/documentation.py

```diff
@@ -0,0 +1,428 @@
+"""
```

Comment:
I am suggesting to change the name of the file to docparser or docstring or docstring_parser. We should have another file in dipy/doc/ about documentation which should talk about the IAP and how it can be used from developers and from users. But this is for much later (in another PR) when this will start stabilizing.